deploy/resource-quotas/README.md

```diff
@@ -0,0 +1,30 @@
+# Handle Resource Quotas
```

Comment:
Will this be noticed?  Probably would put this in the root README instead of in this subdir.

Reply:
I'm not sure? Maybe a pointer in the main readme to this readme?